cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -173,6 +173,81 @@ var _ = Describe("dpdk", func() {
 		}
 	})
 
+	Context("DRAFT - vhost-net", func() {
```

Comment:
From the commands below, you are testing the tap driver.

vhost-net is not involved.
If you want to test virtio-user (which uses the vhost-net backend with a tap), you'll have to call:
--vdev virtio_user0,iface=tap23,path=/dev/vhost-net